src/prompts/create-event-prompt.js

```diff
@@ -2,7 +2,7 @@ export const CREATE_EVENT_PROMPT = `
 Date format: YYYY-MM-DDThh:mm:ss+00:00
 Based on this event description: "Joey birthday tomorrow at 7 pm",
 output a json of the following parameters: 
-Today's datetime on UTC time 2021-05-02T10:00:00+00:00 and timezone
+Today's datetime on UTC time 2021-05-02T10:00:00+00:00, it's Friday and timezone
```

Comment:
Looks like 2nd of May 2021 was a Sunday, should we correct this to avoid risking confusing it?

```suggestion
Today's datetime on UTC time 2021-05-02T10:00:00+00:00, it's Sunday, and timezone
```

Reply:
This has been updated.